Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -473,6 +478,7 @@
         },
         {
             "name": "XmlVariableSubstitution",
+            "aliases": ["enableXmlVariableSubstitution"],
```

Comment:
Can we also have an alias for JSON variable substitution?

Reply:
The aliases are added only if the "name" wasn't intuitive. I believe for Json, we have JsonFiles which was good enough and hence alias is not being added. @RoopeshNair to comment if any.